v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,229 @@
+---
+sidebar_position: 3
+title: Backend Setup
+hide_title: true
+---
+
+# Setting up the Backend
+
+In order to use the anomaly detection service, a request to the following enpoint (based on the region) needs to be made:
+
+**US Region:**
+```typescript
+POST https://security-us-east-1.aws.supertokens.io/v1/security
+```
+
+**EU Region (Ireland):**
+```typescript
+POST https://security-eu-west-1.aws.supertokens.io/v1/security
+```
+
+**APAC Region (Singapore):**
+```typescript
+POST https://security-ap-southeast-1.aws.supertokens.io/v1/security
+```
+
+You can view the HTTP API reference for this endpoint [here](#somelinkhere).
+
+### Retrieving the Request ID
+
+The request ID that has been passed from the frontend is available in the `requestId` property of the input body from each API override.
+
+Below is an example of how to retrieve the request ID from the input body:
+
+```typescript
+const generatePasswordResetTokenPOST = async function (input) {
+    // ... your code here ...
+    const requestId = (await input.options.req.getJSONBody()).requestId;
+    // ... your code here ...
+}
+```
+
+:::important
+The `requestId` should be required when the trying to reset password, sign in or sign up. If the `requestId` is not present, a 400 error should be thrown.
+:::
+
+### Making the Request to the Anomaly Detection endpoint
+
+The request body is a JSON object that contains the following properties (all fields are optional):
+
+- `securityServiceRequestId`: The request ID that has been generated on the frontend. If this is omitted, the bot detection, impossible travel detection, new device detection, device count detection and request ID info will be skipped.
```

Comment:
the frontend uses requestId. Need to correct the frontend

Reply:
yes, but this is the input for our api. Should we change it in the api too ?

Reply:
done